pyuvdata/uvdata/mir_parser.py

```diff
@@ -861,37 +1064,47 @@ def _read_data(self, data_type, return_vis=True, use_mmap=True, read_only=False)
         Returns
         -------
         data_dict : dict
-            A dictionary, whose the keys are matched to individual values of sphid in
-            `sp_data`, and each entry contains a dict with two items. If
-            `return_vis=False` then a "raw data" dict is passed, with keys "scale_fac",
-            an np.int16 which describes the common exponent for the spectrum, and
-            "data", an array of np.int16 (of length equal to twice that found in
-            `sp_data["nch"]` for the corresponding value of sphid) containing the
-            compressed visibilities.  Note that entries equal to -32768 aren't possible
-            with the compression scheme used for MIR, and so this value is used to mark
-            flags. If `return_vis=True`, then a "vis data" dict is passed, with keys
-            "data", an array of np.complex64 containing the visibilities, and
-            "flags", an array of bool containing the per-channel flags of the
-            spectrum (both are of length equal to `sp_data["nch"]` for the
-            corresponding value of sphid).
+            A dictionary, whose the keys are matched to individual header keys -- if
+            `data_type="auto"`, then values of achid in `ac_data` are used; if
+            `data_type="cross"`, then  values of sphid in `sp_data`, are used. Matched
+            to each of these keys is a dict with two items. If the data are "compressed"
+            (like is typical for visibilities) and `scale_data=True`,  then a "raw data"
+            dict is passed, with keys "scale_fac", an np.int16 which describes the
+            common exponent for the spectrum, and "data", an array of typically np.int16
+            (of length equal to twice that found in "nch" for the corresponding metadata
+            container (n.b, the maximum negative interger value, -32768 for int16,
```

Comment:
```suggestion
            container (n.b, the maximum negative integer value, -32768 for int16,
```

Reply:
Fixed in be15abab6f4548a576351a1f43d190cefd2fcd48.